src/platform/switch/cursor.c

```diff
@@ -15,8 +16,7 @@ extern struct {
 
 static SDL_Texture *init_cursor(const cursor *c)
 {
-
-    SDL_Texture *tex = SDL_CreateTexture(SDL.renderer, SDL_PIXELFORMAT_ARGB8888, SDL_TEXTUREACCESS_STREAMING, 32, 32);
+    SDL_Texture *tex = SDL_CreateTexture(SDL.renderer, SDL_PIXELFORMAT_ARGB8888, SDL_TEXTUREACCESS_STATIC, 32, 32);
```

Comment:
the worry i have with this and the vita code is that the below is hardcoded to expect a 32*32 array of cursor data. Instead, e.g., the arrow cursor at scale 1 is 13*21

Reply:
Should be fixed now.